pyproject.toml

```diff
@@ -11,7 +11,8 @@ description-file = "README.md"
 classifiers = ["License :: OSI Approved :: MIT License"]
 requires-python = ">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*"
 requires = [
-  "pathlib2; python_version < '3.4'"
+  "pathlib2; python_version < '3.4'",
+  "six",
```

Comment:
considering the size of this IMHO we should try to make it work without ``six``

Reply:
I would like to make all things work before worrying about dependencies, since the implementation would provide perspective how much the dependencies are useful. This is especially true with six because we are dealing with a lot of file IO, where Python 2 and 3 behave quite differently.

(I do think we’ll eventually get rid of `pathlib2` though. I thought it’s useful at first, but found it a bit awkward to use when dealing with zip entries.)